omod-2.2/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_2/ConditionResource2_2.java

```diff
@@ -59,7 +59,7 @@ public DelegatingResourceDescription getRepresentationDescription(Representation
 			DelegatingResourceDescription description = new DelegatingResourceDescription();
 			description.addProperty("uuid");
 			description.addProperty("display");
-			description.addProperty("condition");
+			description.addProperty("condition", Representation.REF);
```

Comment:
Why not leave Representation.REF on condition for only the default representation?

Reply:
Okay

Reply:
Can you also put Representation.REF for previouseVersion?

Reply:
Should I put it on both representations or just this one?

Reply:
Only the default